common/utils/Clock.cpp

```diff
@@ -129,6 +131,11 @@ int64_t BaseTimeVal::InMilliSeconds() const {
           m_tv.tv_usec / ONE_THOUSAND);
 }
 
+int64_t BaseTimeVal::InMicroSeconds() const {
```

Comment:
This returns exactly the same value as BaseTimeVal::AsInt(), any reason we need the second function?

Reply:
This is returning msec not usec...

Reply:
I guess AsInt is too then; I've just double-checked and the code is identical in both, only the method name changes.